### PR TITLE
#6 Front: auth page (update)

### PR DESCRIPTION
**Pull request by NastyaSvich**
Там есть проблемка с чтением стилей c файла Auth.module.scss (их как-будто не видит файл Auth.tsx)

### CODE REVIEW

**Comment by tritonJS826** [comment]
Problem: PR comment on russian

File: frontend/src/AppRoutes.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+// import Menu from 'Logic/Menu/Menu';
+import Auth from './Logic/Auth/Auth';
 import Options from './Logic/Options/Options';
 import About from './Logic/About/About';
 import Statistic from './Logic/Statistic/Statistic';
 
 const AppRoutes = {
   homePage: new PageMeta<void>({
     path: '/menu',
-    render: () => <Menu />,
```
Comment (tritonJS826) [inline frontend/src/AppRoutes.tsx:11]: Please,return this line back)

File: frontend/src/AppRoutes.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import PageMeta from 'Kit/PageMeta';
-import Menu from 'Logic/Menu/Menu';
```
Comment (tritonJS826) [inline frontend/src/AppRoutes.tsx:2]: Return this line back, pls)

File: frontend/src/Components/ElementsMenu/ElementsMenu.module.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+.menu li:hover .trim {
+    left: 0px;
+    opacity: 1.0;
+}
+
+.menu li:hover svg {
+    opacity: 1.0;
+}
```
Comment (tritonJS826) [inline frontend/src/Components/ElementsMenu/ElementsMenu.module.scss]: Please, use scss syntax (for example nested styles)

File: frontend/src/Components/ElementsMenu/ElementsMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+interface ElementsMenuProps {
+  text: string
+}
+
+/**
+ * ElementsMenu
+ */
+const ElementsMenu: React.FC<ElementsMenuProps> = (props: ElementsMenuProps) => (
+  <div>
```
Comment (tritonJS826) [inline frontend/src/Components/ElementsMenu/ElementsMenu.tsx]: Is this div required?

File: frontend/src/Logic/Auth/Auth.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
+
+import React from 'react';
+import { Login, Register } from './components/index';
+import "./Auth.module.scss";
+
+/**
+ * Auth page
+ */
+export default class Auth extends React.Component {
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.jsx]: It's better to use declarative style with functional components

File: frontend/src/Logic/Auth/Auth.module.scss [hunk 6]
```diff
@@ -0,0 +1,85 @@
+//Custom Fonts
+@import url("https://fonts.googleapis.com/css?family=Open+Sans&display=swap");
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.module.scss]: It's better to import fonts globally

File: frontend/src/Logic/Auth/Auth.module.scss [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+.login {
+  width: 27em;
+  height: 40em;
+  //min-height: 44em;
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.module.scss]: Commented code is forbidden

File: frontend/src/Logic/Auth/Auth.jsx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLogginActive: true,
+    };
+  }
+
+  componentDidMount() {
+    // Add .right by default
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.jsx]: This comment is not required

File: frontend/src/Logic/Auth/Auth.module.scss [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+      font-weight: 500;
+      color: #fff;
+      margin-right: 3em;
+      margin-left: 3em;
+    }
+  }
+}
+
+//Button
+.btn {
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.module.scss]: It's better to use classname button instead .btn and comment

File: frontend/src/Logic/Auth/Auth.jsx [hunk 10]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.jsx]: Why?

File: frontend/src/Logic/Auth/Auth.jsx [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/Auth.jsx]: Why jsx?

File: frontend/src/Logic/Auth/components/index.jsx [hunk 12]
```diff
@@ -0,0 +1,4 @@
+import './style.scss';
+
+export { Login } from './login';
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/components/index.jsx]: Prefer to use absolute pathes

File: frontend/src/Logic/Auth/components/login.jsx [hunk 13]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import React from 'react';
+import loginImg from "./login.svg";
+
+export class Login extends React.Component {
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/components/login.jsx]: Prefer to use functional components

File: frontend/src/Logic/Auth/components/login.svg [hunk 14]
```diff
@@ -0,0 +1 @@
...17.57 728.97 518.57 728.97 509.9 676.72 508.9 676.72 517.57" fill="#3f3d56"/><path d="M534.13,486.79c.08,7-3.16,13.6-5.91,20.07a163.49,163.49,0,0,0-12.66,74.71c.73,11,2.58,22,.73,32.9s-8.43,21.77-19,24.9c17.53,10.45,41.26,9.35,57.76-2.66,8.79-6.4,15.34-15.33,21.75-24.11a97.86,97.86,0,0,1-13.31,44.75A103.43,103.43,0,0,0,637,616.53c4.31-5.81,8.06-12.19,9.72-19.23,3.09-13-1.22-26.51-4.51-39.5a266.06,266.06,0,0,1-6.17-33c-.43-3.56-.78-7.22.1-10.7,1-4.07,3.67-7.51,5.64-11.22,5.6-10.54,5.73-23.3,2.86-34.88s-8.49-22.26-14.06-32.81c-4.46-8.46-9.3-17.31-17.46-22.28-5.1-3.1-11-4.39-16.88-5.64l-25.37-5.43c-5.55-1.19-11.26-2.38-16.87-1.51-9.47,1.48-16.14,8.32-22,15.34-4.59,5.46-15.81,15.71-16.6,22.86-.72,6.59,5.1,17.63,6.09,24.58,1.3,9,2.22,6,7.3,11.52C532,478.05,534.07,482,534.13,486.79Z" transform="translate(-79.34 -172.91)" fill="#3f3d56"/><path d="M379.6,506.5H280a.5.5,0,0,1,0-1h99.6a.5.5,0,0,1,0,1Z" transform="translate(-79.34 -172.91)" fill="#f2f2f2"/><path d="M379.6,515.5H280a.5.5,0,0,1,0-1h99.6a.5.5,0,0,1,0,1Z" transform="translate(-79.34 -172.91)" fill="#f2f2f2"/><path d="M379.6,524.5H280a.5.5,0,0,1,0-1h99.6a.5.5,0,0,1,0,1Z" transform="translate(-79.34 -172.91)" fill="#f2f2f2"/></svg>
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/components/login.svg]: Let's save our images, SVG and other content-files in special folder on highest level

File: frontend/src/Logic/Auth/components/register.jsx [hunk 15]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
```
Comment (tritonJS826) [inline frontend/src/Logic/Auth/components/register.jsx]: ?

File: frontend/src/Logic/Game/Level/GameUIInterface/GameStatistic/GameStatistic.tsx [hunk 16]
```diff
@@ -6,11 +6,8 @@ interface IGameStatistic {
 /*
  * Game statistic page
  */
-const GameStatistic: React.FC<IGameStatistic> = (props: IGameStatistic) => {
```
Comment (tritonJS826) [inline frontend/src/Logic/Game/Level/GameUIInterface/GameStatistic/GameStatistic.tsx:9]: It's better to use GameStatisticProps props

File: frontend/src/Logic/Menu/Menu.css [hunk 17]
```diff
@@ -46,24 +46,21 @@ body {
     opacity: 0.0;
 }
```
Comment (tritonJS826) [inline frontend/src/Logic/Menu/Menu.css]: Css forbidden

File: frontend/src/Logic/Menu/Menu.tsx [hunk 18]
```diff
@@ -1,25 +1,23 @@
...
-// import ElementsMenu from '../../Components/ElementsMenu/ElementsMenu';
-// import infoPages from '../Pages';
-
+import ElementsMenu from 'Components/ElementsMenu/ElementsMenu';
 import './Menu.css';
 
-// interface IMenu {
+// interface MenuProps {
 
 // }
```
Comment (tritonJS826) [inline frontend/src/Logic/Menu/Menu.tsx]: Commented code

File: frontend/src/Logic/Auth/Auth.jsx [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+import React from 'react';
+import { Login, Register } from './components/index';
+import "./Auth.module.scss";
```
Comment (NastyaSvich) [inline frontend/src/Logic/Auth/Auth.jsx]: Dont use double quotes

**Review by tritonJS826 — COMMENTED** [review]
Fix all problems, pls)

**Review by tritonJS826 — DISMISSED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**